lib/AST/Decl.cpp

```diff
@@ -2715,6 +2715,15 @@ void ClassDecl::addImplicitDestructor() {
   // Wire up generic environment of DD.
   DD->setGenericEnvironment(getGenericEnvironmentOfContext());
 
+  // Do the synthetic-destructor-relevant parts of swift::markAsObjcC:
```

Comment:
Typo: markAsObjcC. This function is also private to Sema, though, so it doesn't make sense to reference it here. It's also probably okay to leave off the registration with the parent source file—no one's going to call `deinit` through `AnyObject`.